apicast/src/executor.lua

```diff
@@ -0,0 +1,59 @@
+local resty_env = require('resty.env')
+
+if resty_env.get('APICAST_MODULE') then
+  return require('module')
```

Comment:
Maybe we should do this in the apicast module itself? To keep the compatibility with the old module system. Basically allow people to replace the APIcast module. But we need to figure out how to allow them to require it but to outside return their module. 
Maybe this belongs to the policy_chain.build where it defaults to the apicast module?

Reply:
How about doing this in the `local_chain` module?

When building the local chain, the default is `require('policy_chain').build({ 'apicast' })`. I think we could check the env 'APICAST_MODULE' and do `require('policy_chain').build({ 'module' })` instead if it's set.

Reply:
Hmmmm. That sounds good 👍